### PR TITLE
User roles API 

### PR DESCRIPTION
**Pull request by HarishGangula**

### CODE REVIEW

File: server/config/constants.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ module.exports = {
        "TELEMMETRY": "/action/data/v3/telemetry",
        "FORM_READ": "/action/data/v1/form/read",
        "USER_SEARCH": "action/user/v1/search/*",
-       "USER_ROLE": "/api/role",
-       "USERS": "/api/users",
+       "USER_ROLE": "/api/user/v1/role",
```
Comment (vinukumar-vs) [inline server/config/constants.js:32]: What about front-end changes for this? 

@vaibhavbhuva of these changes are done, share the PR.

**Review by vinukumar-vs — APPROVED** [review]

File: server/config/constants.js (hunk 1 above)
Comment (vaibhavbhuva) [reply server/config/constants.js:32]: @HarishGangula Please check

Comment (HarishGangula) [reply server/config/constants.js:32]: @vaibhavbhuva  This is fine

Comment (vaibhavbhuva) [reply server/config/constants.js:32]: @HarishGangula What about front-end changes for this?  We need to update the same path in the fronted application  also right ?

**Pull request merged**